### DISCUSSION TITLE
Custom delays set for different chords.

### DISCUSSION THREAD

**Discussion by phantom-Orange** [psoukie/zipchord#162] [Ideas]
is it possible to allow each chord to have a custom detection delay? This way you could have multiple outputs from the same input, just set to different delays, for example `the` and `there` could both be set to `th` with different delays. Or chords that are often accidentally triggered could be set to a longer delay whilst the rest of the keys would use the default delay. 
eg: 

`th    the *70`
`th    there *130`

where the *indicates the delay of the chord. any chord not labelled will fall back to the default delay set. 
So at first the result would be `the` but 60ms after that it would expand to `there`.

**Comment by psoukie** [comment]
This is an interesting suggestion! I've added a feature where you can define compound chords, which might be faster than holding the chord. But I can also see how that's too complicated for very short words like 'there.'

```
el	well
el mn	element
```
